components/raftstore-v2/src/fsm/peer.rs

```diff
@@ -127,6 +129,18 @@ impl<'a, EK: KvEngine, ER: RaftEngine, T: Transport> PeerFsmDelegate<'a, EK, ER,
         Self { fsm, store_ctx }
     }
 
+    #[inline]
+    fn schedule_pending_ticks(&mut self) {
+        let pending_ticks = self.fsm.peer.take_pending_ticks();
+        for tick in pending_ticks {
+            if tick == PeerTick::ReactivateMemoryLock {
+                self.register_reactivate_memory_lock_tick()
```

Comment:
what's the difference of register_reactivate_memory_lock_tick and self.schedule_tick(ReactivateMemoryLock)? if it's same, why bother to differentiate it here?

Reply:
Good suggestion.